scripts/build.js

```diff
@@ -168,3 +168,45 @@ function getDisabledTasks(process, defaultDisabled = []) {
 
   return defaultDisabled;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+//// Artifact helper functions
+////////////////////////////////////////////////////////////////////////////////////////////////////
+/**
+   * Gets the size of the "default" locale bundle file
+   */
+function executeTask() {
```

Comment:
should not be called executeTask

Reply:
I'd call this something like `generateSizeData`